Clients/src/presentation/pages/SettingsPage/Organization/index.tsx

```
@@ -127,6 +212,10 @@ const Organization = () => {
         body: "The organization was created successfully.",
         isToast: false,
       });
+
+      if (response && response.id && organizationLogo) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)